aspnet/tutorials/nano-server.rst

```diff
@@ -1,8 +1,149 @@
+
+Installing the HTTPPlatformHandler Module
+-----------------------------------------
+
+At the moment, the process to install the HTTPPlatformHandler Module for IIS is manual. You will need to install the latest 64-bit version of the `HttpPlatformHandler <http://www.iis.net/downloads/microsoft/httpplatformhandler>`_ on a regular (not Nano) machine. After installing you will need to copy the following files:
+
+* *%windir%\\System32\\inetsrv\\httpPlatformHandler.dll*
+* *%windir%\\System32\\inetsrv\\config\\schema\\httpplatform_schema.xml*
+
+On the Nano machine you’ll need to copy those two files to their respective locations.
+
+.. code:: ps1
+
+  Copy-Item .\httpPlatformHandler.dll c:\Windows\System32\inetsrv
+  Copy-Item .\httpplatform_schema.xml c:\Windows\System32\inetsrv\config\schema
+
+Enabling the HTTPPlatformHandler
+--------------------------------
+
+You can execute the following PowerShell script in a remote PowerShell session to enable the HTTPPlatformHandler module on the Nano server.
+
+.. note:: This script runs on a clean system, but is not meant to be idempotent. If you run this multiple times it will add multiple entries. If you end up in a bad state, you can find backups of the *applicationHost.config* file at *%systemdrive%\inetpub\history*.
+
+.. literalinclude:: nano-server/enable-platformhandler.ps1
+  :language: ps1
+
+Manually Editing *applicationHost.config*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can skip this section if you already ran the PowerShell script above. Though is not recommended, you can also enable the HTTPPlatformHandler by manually editing the *applicationHost.config* file.
+
+Open up *C:\\Windows\\System32\\inetsrv\\config\\applicationHost.config*
+
+Under ``<configSections>`` add
+
+.. literalinclude:: nano-server/applicationHost.config
+  :language: xml
+  :lines: 42-43
+  :emphasize-lines: 2
+
+In the ``system.webServer`` section, unlock handlers
+
+.. literalinclude:: nano-server/applicationHost.config
+  :language: xml
+  :lines: 55,63
+  :emphasize-lines: 2
+
+Add ``HTTPPlatformHandler`` to the ``globalModules`` section
+
+.. literalinclude:: nano-server/applicationHost.config
+  :language: xml
+  :lines: 210,224
+  :emphasize-lines: 2
+
+Additionally, add ``HTTPPlatformHandler`` to the ``modules`` section
+
+.. literalinclude:: nano-server/applicationHost.config
+  :language: xml
+  :lines: 275,286
+  :emphasize-lines: 2
+
+Installing an ASP.NET 5 application
+-----------------------------------
+
+I have copied the published contents of my application to *C:\\HelloAspNet5\\* on the Nano server. Let's create a new site in IIS to point to the output of our published app using this PowerShell snippet. We are using the ``Default App Pool`` in this example for simplicity. For more considerations on running under an AppPool, refer to the :ref:`apppool` doc.
```

Comment:
> Let's create a new site in IIS to point to the output of our published app using this PowerShell snippet. We are using the ``Default App Pool`` in this example for simplicity. For more considerations on running under an AppPool, refer to the :ref:`apppool` doc. 

Use the following PowerShell snippet to create a new site in IIS for our published app. This script uses the ``Default App Pool`` for simplicity. For more considerations on running under an AppPool, see :ref:`apppool`.